doc/man7/migration_guide.pod

```diff
@@ -467,6 +467,18 @@ Remove your usage of the low level APIs. In this case you will need to rewrite y
 
 =back
 
+=head3 Error code changes
+
+As OpenSSL 3.0 provides a brand new Encoder/Decoder mechanism for working with
+widely used file formats, an application code that checks for particular error
+reason codes on key loading failures might need an update.
+
+The password-protected keys may deserve a special attention. If only some errors
+are treated as an indicator that the user should be asked about the password again,
+it's worth adding processing the error lib B<ERR_LIB_OSSL_DECODER>.
```

Comment:
Actually I am not sure this is right advice as with current master you should not get ERR_LIB_OSSL_DECODER in error code if you just use wrong password. Unfortunately I do not have a better one that would be comprehensive. It really depends on what the application is doing.

Reply:
```suggestion
it's worth adding processing for the error lib B<ERR_LIB_OSSL_DECODER>.
```

Reply:
If I replace ```it's worth adding processing the error lib B<ERR_LIB_OSSL_DECODER>.``` with 
```it's worth test these scenarios and start processing the relevant codes```, is it OK?

Reply:
Yes, that would be better. With `worth to test` and `processing the newly relevant codes`

Reply:
I'd fix the grammar as `it's worth testing these scenarios and processing the newly relevant codes` but agree that the proposed formulation addresses t8m's concern